djangocms_versioning/admin/using_versioning.py

```diff
@@ -0,0 +1,222 @@
+from django.conf.urls import url
```

Comment:
I'm not so sure about the filename for this file, following where this is in the cms I would opt for somethign similar but generic, maybe admin/objectadmin.py or genericadmin.py

https://github.com/divio/django-cms/blob/release/4.0.x/cms/admin/pageadmin.py